CMakeLists.txt

```diff
@@ -337,7 +337,9 @@ IF(NOT CYCLUS_DOC_ONLY)
 
         # make sure we know about having python
         add_definitions(-DCYCLUS_WITH_PYTHON)
-        set(LIBS ${LIBS} ${PYTHON_LIBRARIES})
+        IF (NOT APPLE)
+            set(LIBS ${LIBS} ${PYTHON_LIBRARIES})
```

Comment:
I guess another question - do we need to add `PYTHON_LIBRARIES` for linux.  I guess it's clear we don't need it for MacOS, but could we do without it for Linux?

Reply:
That is, linux may tolerate it, but it still may not be necessary??? (pure speculation...)

Reply:
I'll try this

Reply:
this is required for `linux`